srcpkgs/python3-TxSNI/template

```diff
@@ -1,21 +1,19 @@
 # Template file for 'python3-TxSNI'
 pkgname=python3-TxSNI
-version=0.1.9
-revision=4
-create_wrksrc=yes
-build_wrksrc="TxSNI-${version}"
+version=0.2.0
+revision=1
+wrksrc="txsni-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-Twisted python3-openssl"
+checkdepends="python3-devel"
```

Comment:
Just drop this, the tests fetch a bunch of prebuilt wheels and don't actually check that our package works.

If the license file is now included in the PyPI tarball, prefer PyPI; otherwise keep the GitHub tarball to avoid fetching the license separately.

Reply:
Pypi version has no license